index.rb

```diff
@@ -144,6 +146,31 @@ def show(**kwargs)
   end
 end
 
+venues << Venue.new(:name => 'Knockout', :link => 'https://theknockoutsf.com') do
+  URI.open(URI.join(link, 'events/feed')) do |xml|
+    Nokogiri::XML(xml).css('item').map do |item|
+      # Nokogiri doesn't decode this correctly, so we have to force it :(
```

Comment:
I used the built-in RSS gem for Bottom of the Hill: https://github.com/davishmcclurg/shows/blob/59b7e84b8dd634d7d97237dbb35c739b379369d7/index.rb#L52

Would that work here so that we don't have Nokogiri issues?

Reply:
I swapped this out to the `rss` gem, but the descriptions still have html-encoded bits present.

I checked in a workaround where we html parse the content encoded description of each item, and then just show the text without any html decorations as the description, which seems to work alright, and has less risk of us embedding someone else's arbitrary html.